src/lucky_avram/ext/lucky/input_helpers.cr

```diff
@@ -211,6 +212,7 @@ module Lucky::InputHelpers
       "value" => input_value(field),
     }.merge(input_overrides)
     update_array_id_counter!(field)
+    attrs.uniq!
```

Comment:
just in case so you don't accidentally send over duplicate attrs...

Reply:
HTML is pretty fast and loose with it's attributes, especially in the age of React and other front end frameworks which make up their own elements...are we sure there is no use case for passing the same attribute more than once to a tag?

Reply:
This would only prevent a tag being rendered like `<input ... checked checked required required />`. I'm sure browsers would just ignore it, so it's probably not required, but in the case of some random browser that breaks due to it, I figured why not? 🤷‍♂️